workspace.json

```diff
@@ -3403,6 +3403,11 @@
                 "glob": "migrations/**",
                 "input": "apps/gjafakort/application",
                 "output": "./"
+              },
+              {
+                "glob": ".sequelizerc",
```

Comment:
The glob above should cover this: `"*sequelize*"`???

Reply:
It doesn't catch hidden files. Might be something that could be fixed in the esbuildnx package but this is an ok fix for now IMO.